blockly-ws-search/example/web-imports/dist/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
I thought dist was for built files, and therefore gitignored.

Reply:
We still need this index file so just changing this to be public and then copying over this into the build folder when we run webpack.